### ISSUE TITLE
Use default theme

### ISSUE THREAD

**Issue by jkaberg** [ezand/lovelace-posten-card#3]
Hi @ezand, thanks for this fine addon - using it daily. 

One wish from me is if you could adher to the theme standard instead of whats currently going on (where white background is forced)? Heres an pic to show you what I mean

![image](https://user-images.githubusercontent.com/613363/103616022-f5259100-4f2b-11eb-9f3f-8907ebb53eb5.png)

The posten card kind of standsout, while it doesnt need to (IMHO). Perhaps make it optional?

**Comment by ezand** [comment]
I totally agree @jkaberg. This is actually in my plans to do when I get the time :-) This was kind of my first attempt on a Lovelace card, to experiment. So it definitely needs some improvements. I think I will make most part of it customizable, including the theme 👍

**Comment by elgeniskogen** [comment]
I have added the Posten Card with HACS, but how do I add it to the Lovelace UI? It is not on the list of cards. Do I have to use the manual card? What is then the code to use? Appreciate an example :-)

**Comment by Twsts** [comment]
Bump on support for dark/night mode.
Anyways - thanks for your efforts!
